lib/public/models/BkpRolesSelectionModel.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { SelectionDropdownModel } from '../components/common/selection/dropdown/SelectionDropdownModel.js';
+import { dplDetectorsProvider } from '../services/detectors/dplDetectorsProvider.js';
+import { getRoleForDetector } from '../utilities/getRoleForDetector.js';
+
+const DEFAULT_ROLES_OPTIONS = [{ value: 'admin' }, { value: 'guest' }];
+
+/**
+ * Model storing state of selected Bkp roles (for TEST only)
+ */
+export class BkpRolesSelectionModel extends SelectionDropdownModel {
```

Comment:
probably no need to prefix by bookkeeping, or if you want you can prefix by user `UserRoleSelectionModel`

Reply:
I'd say either we leave this or change name of enum in `BkpRoles.js`

Reply:
It's different. BkpRoles list the roles of bkp application, here you chose the roles of the current user

Reply:
applied

Reply:
You also need to rename the variables

Reply:
renamed